### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Alhajideen**
## To do app
In this milestone, I was able to;
- Clone my webpack broiler place and customized it for a fresh project
- Create a class for the Todo list
- Create an array to hold class objects
- Get data from the form and use it to create a new object
- Create a function to iterate over the array and display the components to dom.

I anticipate your feedback 😁

### CODE REVIEW

**Review by addod19 — COMMENTED** [review]
### Hi @Alhajideen, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is failing the linter checks

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+btn.addEventListener('click', () => {
+  const id = Date.now();
+  const desc = document.querySelector('.description');
+  const description = desc.value;
+  const completed = false;
+  const newTodo = new Todo(id, description, completed);
+  items.push(newTodo);
+  addtoDom();
+});
+
```
Comment (Reem-lab) [inline src/index.js]: - [ ] I think you miss understanding the requirements, they are about  to make a static structure  for the todolist so please check the requirements and implement it as required, I know that you made a huge efforts with adding dynamically but the required is just to display some tasks and what you did now this is for the next milestone 💪🏻💯💥

The last required is a function just to display the tasks from the array 💥💯💪🏻

image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
#Hi @Alhajideen, 👋🏻👋🏻

Good job so far! 💥💥💯💪🏻💪🏻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the design✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Alhajideen 👏

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**